src/preparers.py

```diff
@@ -199,6 +199,35 @@ def prepare_open_assistant(dset):
     return messages
 
 
+def prepare_open_assistant_v2(dset):
```

Comment:
Is this different at all from the original? https://github.com/Data-Provenance-Initiative/Data-Provenance-Collection/blob/main/src/preparers.py#L173C8-L173C27

perhaps they could both call the same function?

Reply:
Yeah, they both can call the same function